docs/core/tutorials/creating-app-with-plugin-support.md

```diff
@@ -8,13 +8,16 @@ ms.date: 10/16/2019
 
 # Create a .NET Core application with plugins
 
-This tutorial shows you how to create a custom <xref:System.Runtime.Loader.AssemblyLoadContext> to load plugins. An <xref:System.Runtime.Loader.AssemblyDependencyResolver> is used to resolve the dependencies of the plugin. The tutorial correctly isolates the plugin's dependencies from the hosting application. You'll learn how to:
+This tutorial shows you how to create a custom <xref:System.Runtime.Loader.AssemblyLoadContext> to load plugins. An <xref:System.Runtime.Loader.AssemblyDependencyResolver> is used to resolve the dependencies of the plugin. The tutorial provides a separate namespace for the plugin's dependencies, allowing different assembly dependencies between the plugins and the hosting application. You'll learn how to:
 
 - Structure a project to support plugins.
 - Create a custom <xref:System.Runtime.Loader.AssemblyLoadContext> to load each plugin.
 - Use the <xref:System.Runtime.Loader.AssemblyDependencyResolver?displayProperty=fullName> type to allow plugins to have dependencies.
 - Author plugins that can be easily deployed by just copying the build artifacts.
 
+> [!NOTE]
+> .NET does not provide a security or reliability boundary beyond the operating system's process boundary. Untrusted code cannot be safely loaded into a trusted .NET process. To provide a security or reliability boundary, consider a technology provided by your OS or virtualization platform. 
```

Comment:
I don't think the term "reliability" is appropriate here as the immediate next question is "reliability of what?"

Reply:
```suggestion
> Untrusted code cannot be safely loaded into a trusted .NET process. To provide a security or reliability boundary, consider a technology provided by your OS or virtualization platform. 
```

I think removing the first sentence might actually be sufficient.